integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiServiceMigration.java

```diff
@@ -206,125 +217,113 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
   }
 
   /**
-   * Verify JMS Service is migrated to an available active server.
-   * Here the JMS messages are stored in Filestore on PV
+   * Verify JMS/JTA Service is migrated to an available active server.
    */
   @Test
-  @DisplayName("Verify JMS Service migration with FileStore")
-  void testMiiJmsServiceMigrationWithFileStore() {
+  @DisplayName("Verify JMS/JTA Service migration with File(JDBC) Store")
+  void testMiiJmsJtaServiceMigration() {
 
-    // build the standalone JMS Client on Admin pod after rolling restart
+    // build the standalone JMS Client on Admin pod 
     String destLocation = "/u01/JmsSendReceiveClient.java";
     assertDoesNotThrow(() -> copyFileToPod(domainNamespace,
         adminServerPodName, "",
         Paths.get(RESOURCE_DIR, "jms", "JmsSendReceiveClient.java"),
         Paths.get(destLocation)));
     runJavacInsidePod(adminServerPodName, domainNamespace, destLocation);
 
-    assertTrue(checkJmsServerRuntime("managed-server2"),
-         "JMSService@managed-server2 is on managed-server2 before migration");
+    assertTrue(checkJmsServerRuntime("ClusterJmsServer@managed-server2", 
+         "managed-server2"),
+        "ClusterJmsServer@managed-server2 is on managed-server2 before migration");
```

Comment:
If its false, shouldn't the message be ClusterJmsServer@managed-server2 is not on managed-server2 before migration.

Reply:
done